deploy/main.py

```diff
@@ -90,6 +89,3 @@
 
 #core.setDefaultLanguage("en") #set default language!
 app = core.setup(modules, render)
-
-if __name__ == "__main__":
```

Comment:
Are you sure this will work?

Reply:
Yes, `print(__name__)` prints `main`, so this branch will never be entered. And there is no `run`-method in the core

Reply:
Umm okay.. I didn't know about that. Then its okay.